subprojects/packagefiles/taskflow/meson.build

```diff
@@ -0,0 +1,32 @@
+project('taskflow', 'cpp',
+    version: '3.4.0',
+    license: 'MIT',
+)
+
+# Header-only library
+taskflow_inc = include_directories('.')
```

Comment:
you need either more than this or edit taskflow_dep. this is why your tests are failing.

Reply:
That is, add the taskflow directory to your includes.

Reply:
Oh, I mean this line: https://github.com/mesonbuild/wrapdb/pull/515/files#diff-beaae08280a916e55d3e2988e2ffd4b697763330d05c6fc0452cd638d8bff5fdR30

Does it look like code smell to you? If you are okay with that, I can go on adding the `-Dtest=false` option to meson_option.txt.